test/amiadapters/test_sentryx.py

```diff
@@ -0,0 +1,279 @@
+import datetime
+from unittest import mock, TestCase
+
+from amiadapters.base import GeneralMeter, GeneralMeterRead
+from amiadapters.config import AMIAdapterConfiguration
+from amiadapters.sentryx import SentryxAdapter, SentryxMeter, SentryxMeterRead, SentryxMeterWithReads
+
+
+class MockResponse:
+        def __init__(self, json_data, status_code):
+            self.json_data = json_data
+            self.status_code = status_code
+
+        def json(self):
+            return self.json_data
+
+
+def mocked_get_devices_response_first_page(*args, **kwargs):
+    data = {
+        "meters": [
+            {
+                "dmaObjectId": None,
+                "dmaName": None,
+                "deviceId": 654419700,
+                "isDisconnectableDevice": False,
+                "serviceStatus": "NotRDM",
+                "deviceStatus": "OK",
+                "street": "10327 SW SHAD RD, ",
+                "city": "CROOKED RIVER RANCH                               ",
```

Comment:
@mdowell12 is this a real address? I think we should avoid real addresses if possible when hardcoding test cases etc. throughout the code to avoid any semblance of PII.